src/validators/__tests__/isValidCurrency.test.js

```diff
@@ -0,0 +1,18 @@
+import isValidCurrency from 'validators/isValidCurrency'
+
+test('Returns false if currency is invalid', () => {
+  expect(isValidCurrency(undefined)).toBe(false)
+  expect(isValidCurrency(1234)).toBe(false)
+  expect(isValidCurrency('1.123,50323 €')).toBe(false)
+  expect(isValidCurrency('1,000,00 €')).toBe(false)
+  expect(isValidCurrency('1000€')).toBe(false)
```

Comment:
@JoaoFGuiomar i think we should allow this one

Reply:
[Source](http://publications.europa.eu/code/pt/pt-370303.htm)
<img width="567" alt="screenshot" src="https://user-images.githubusercontent.com/2249046/39930728-63fa6ef2-5533-11e8-9344-e73724653aa3.png">

Reply:
@JoaoFGuiomar well can't argue with that 😂